### DISCUSSION TITLE
[Feature Request] Show data labels on time series visualizations

### DISCUSSION THREAD

**Discussion by lucaswhornsby** [grafana/grafana#58490] [Dashboards, Panels & Visualizations]
We would like to have the option to enable data labels for time series. For example, other panel types like bar graphs display the value above each bar. However, when looking at a line graph, the only way to see the Y value of points on the graph is to hover over it to open the tooltip. However, we would like to be able to show the labels above the line automatically. Seeing the Y coordinates of points on a line graph will considerably improve readability and expedite our journey to insights. 

See attached screenshots from Power BI for examples (please disregard mark ups used for redactions for privacy)

![time series PBI](https://user-images.githubusercontent.com/113460677/200700650-91aac96b-1798-47d4-99c1-0a62f138aec1.png)
![time series PBI 2](https://user-images.githubusercontent.com/113460677/200700652-46fc9b5f-7aa5-48ae-aefd-8be312942f76.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
